packages/query/src/graphql/graphql.test.ts

```diff
@@ -176,4 +189,52 @@ describe('GraphqlModule', () => {
 
     expect(fetchedMeta).toMatchObject(mock);
   });
+
+  // sum(price_amount)
+  it('AggregateSpecsPlugin support big number', async () => {
```

Comment:
I think we need a test for number as well to make sure we're not changing the behaviour there

Reply:
Changes have been made.